pkg/scyllaclient/config_client.go

```diff
@@ -209,6 +209,18 @@ func (c *ConfigClient) UUIDSStableIdentifiers(ctx context.Context) (bool, error)
 	return resp.Payload, err
 }
 
+// ConsistentClusterManagement returns true if node uses RAFT for cluster management and DDL.
+func (c *ConfigClient) ConsistentClusterManagement(ctx context.Context) (bool, error) {
+	resp, err := c.client.Config.FindConfigConsistentClusterManagement(config.NewFindConfigConsistentClusterManagementParamsWithContext(ctx))
+	if isStatusCode400(err) {
+		return false, nil
+	}
```

Comment:
What is the meaning of HTTP 400 here ? Current scylla doesn't support this config param ?

I've found this swagger doc in enterprise repo https://github.com/scylladb/scylla-enterprise/blob/enterprise/api/api-doc/config.json

There is nothing about 400 code.

Reply:
> What is the meaning of HTTP 400 here ? Current scylla doesn't support this config param ?

I think that it is used for backward compatibility. Without that, checking Scylla config `consistent_cluster_management` returns errors in Scylla versions that don't have this field introduced. Since SM 3.2 still supports Scylla 5.1 (restore compatibility matrix), there is still a need for this small workaround.